docs/about/CONTRIBUTORS.md

```diff
@@ -1,10 +1,17 @@
 # Contributors
 
-* [Stephanie (Stimac) Drescher ![image](https://avatars1.githubusercontent.com/u/18073131?v=3&s=100)](https://github.com/ststimac)
-* [David García ![image](https://avatars1.githubusercontent.com/u/1581288?v=3&s=100)](https://github.com/sarvaje)
-* [Catalin Maris ![image](https://avatars2.githubusercontent.com/u/1223565?v=3&s=100)](https://github.com/alrra)
-* [Antón Molleda ![image](https://avatars2.githubusercontent.com/u/606594?v=3&s=100)](https://github.com/molant)
-* [Qing Zhou ![image](https://avatars3.githubusercontent.com/u/20218531?v=3&s=100)](https://github.com/qzhou1607)
+<!-- Contributors START
```

Comment:
We will customize things, so we will most probably not be using the [CLI](https://www.npmjs.com/package/all-contributors-cli). That being said, this comment should be removed. Having it just introduces another place where the information needs to be updated.